document/document_test.go

```diff
@@ -510,6 +526,49 @@ func TestNewFromStruct(t *testing.T) {
 		require.NoError(t, err)
 		require.EqualValues(t, 2, v.V.(int64))
 	})
+
+	t.Run("Embedded struct", func(t *testing.T) {
+		db, err := genji.Open(":memory:")
```

Comment:
I think that we can avoid using a real database for testing this. What about using the `GetByField` function just above?

Reply:
`GetByField` is used https://github.com/genjidb/genji/blob/47bae6e2b7a6c1de9277e3114a5a3038879806b0/document/document_test.go#L563

I did it because at first when I wrote my tests, I did used the issue example after some work I deleted queries `Insert` and `Select`... and I forgave to delete the `create` statement and `open`.
Indeed we should work only with the returned `document` and its methods.
Going to delete.